src/js/modules/main/main-view-buttons.js

```diff
@@ -0,0 +1,25 @@
+import Button from '../components/button';
+
+function createButton(label, eventHash, eventContext) {
+  const instance = new Button({
+    modifier: 'accept',
+    visible: false,
+    label });
+
+  for (const event in eventHash) {
+    if (!eventHash[event].hasOwnProperty()) {
```

Comment:
what `hasOwnProperty` is going to do here? from specification (https://tc39.github.io/ecma262/#sec-hasownproperty) is should have one parameter - attribute name.